packages/swift/Sources/Generated.swift

```diff
@@ -128,7 +128,7 @@ public enum AccountEvent {
     }
 }
 
-public enum ConnectLoadedInitialStep: Codable {
+public enum ConnectLoadedInitialStep: String, Codable {
```

Comment:
Technically unrelated but I think I can sneak this one in. Fixes the decoding of this enum value since Swift uses integer based enums by default and we pass back a string value in our post messages.